psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -1432,17 +1432,8 @@ void OEProp::compute_mo_extents() {
 }
 
 std::vector<SharedVector> MultipolePropCalc::compute_mo_extents(bool print_output) {
-    std::shared_ptr<Molecule> mol = basisset_->molecule();
-    SharedMatrix Ca;
-    SharedMatrix Cb;
 
-    if (same_orbs_) {
-        Ca = Ca_ao();
-        Cb = Ca;
-    } else {
-        Ca = Ca_ao();
-        Cb = Cb_ao();
-    }
+    SharedMatrix Ca = Ca_ao();
 
     std::vector<SharedVector> mo_es;
     mo_es.push_back(SharedVector(new Vector("<x^2>", basisset_->nbf())));
```

Comment:
Could you convert to `std::make_shared<Vector>("<x^2>", basisset_->nbf())`? I am actually surprised the construct using `new` is still around...

Reply:
Done!